### ISSUE TITLE
MS Excel 2016 / 365: After editing a cell, NVDA is sluggish for some seconds while navigating through the cells

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9316]
### Steps to reproduce:
1. Open a blank excel sheet in ms Excel 2016 / 365
2. In Excel's options enable "allow direct editing in cells" (works only in NVDA alpha-16813,a0b991af and later)
3. In cell a1 type =1+2 or anything else and press enter

### Actual behavior:
NVDA is very sluggish after pressing enter. Navigating between cells is very slow for 5 seconds or so.

### Expected behavior:
NVDA should be as performant as before editing the cell.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-16813,a0b991 af

#### Windows version:
Windows 10 1809 Update

#### Name and version of other software in use when reproducing the issue:
MS Excel 2016, MS Excel 365

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
No, Alphas or official releases before this one does not work with editing in cells enabled.

**Comment by Adriani90** [comment]
Input before editing cell:

```
Input: kb(laptop):rightArrow
IO - inputCore.logTimeSinceInput (14:53:42.667):
0.096 sec since input
IO - speech.speak (14:53:42.668):
Speaking [u'C4']
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (14:53:42.671):
oleacc.AccessibleObjectFromEvent with window 658010, objectID 634 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (14:53:42.673):
oleacc.AccessibleObjectFromEvent with window 658010, objectID 635 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (14:53:42.674):
oleacc.AccessibleObjectFromEvent with window 658010, objectID 637 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (14:53:42.859):
Input: kb(laptop):rightArrow
IO - inputCore.logTimeSinceInput (14:53:42.940):
0.082 sec since input
```

Input after pressing enter to leave the edit field of the cell:
```
Input: kb(laptop):downArrow
IO - inputCore.logTimeSinceInput (14:53:50.105):
1.003 sec since input
IO - speech.speak (14:53:50.105):
Speaking [u'C4']
```

**Comment by Adriani90** [comment]
This behavior is extremely annoying when you must edit many cells in a short time range. To be honnest, I prefer to disable direct editing in cells like before because the delay is really significant.

**Comment by Adriani90** [comment]
In 2019.1 Beta 1 this does not occur anymore, everything is fine so far. Closing as works for me.
